lib/rollbax.ex

```diff
@@ -30,8 +30,7 @@ defmodule Rollbax do
 
   def report(exception, stacktrace, meta \\ %{} , occurr_data \\ %{})
   when is_list(stacktrace) and is_map(meta) and is_map(occurr_data) do
-    message = Exception.format(:error, exception, stacktrace)
-    meta = Map.put(meta, :rollbax_occurr_data, occurr_data)
-    Rollbax.Client.emit(:error, message, meta)
+    Rollbax.Client.emit(:error, exception, stacktrace,
+                        Map.put(meta, :rollbax_occurr_data, occurr_data))
```

Comment:
Let's keep the `meta = ...` line here instead of building the metadata on the fly; this is refactoring, and I'd prefer the refactoring to happen in a separate PR (if we really need it) because otherwise the behavioural changes get buried a bit :).